client/components/ViewChecks.jsx

```diff
@@ -71,43 +86,55 @@ export default class ViewChecks extends React.Component {
   }
 
   render() {
+    const { classes } = this.props;
     const { url } = this.props;
 
     const checks = this.state.openChecks.map(check => (
       <Grid item xs={12} key={check.checkId}>
         <Grid container spacing={1}>
           <Grid item xs={6}>
-            <Typography component="div">
-              <Box fontWeight="fontWeightBold">
-                Check #{check.checkId}
-              </Box>
+            <Typography variant="h4" noWrap color="primary" className={classes.table}>
+              Table {check.tableId}
+            </Typography>
+            <Typography variant="h5" className={classes.time}>
               {this.parseTime(check.createdAt)}
             </Typography>
           </Grid>
-          <Grid item xs={5}>
-            <Typography align="right" style={{ marginTop: '-5px' }}>
-            Table {check.tableId}<br></br>
-              <Button size="small" variant="contained" style={{ backgroundColor: '#118AB2', color: 'white', fontSize: '10px', marginBottom: '10px', borderRadius: '0px', padding: '2px 8px' }}>
-                <Link to={`${url}/checkitem/${check.checkId}/${check.tableId}`}>
-            View Check
-                </Link>
-              </Button>
-            </Typography>
+          <Grid item xs={6}>
+            <Box display="flex" justifyContent="flex-end" flexWrap="wrap">
+              <Box>
+                <Typography align="right" variant="h5" noWrap className={classes.checkId}>
+                  Check #{check.checkId}
+                </Typography>
+              </Box>
+              <Box>
+                <Button
```

Comment:
Link needs to be wrapping button, not button wrapping link

Reply:
See the original code, it was written by me.

Reply:
alright, I changed this on my next PR anyway